app/views/posts/show.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Placeholder for /users/:id/posts/:post_id</h1>
\ No newline at end of file
+<div class="post">
+  <div class="post-card">
+    <h2 class="post-header">
+      Post <%= @post.id %> by <%= @post.author.name %>
+    </h2>
+    <div class="post-body">
+      <%= @post.text %>
+    </div>  
+    <div class="comments-card">
+      <% @post.comments.each do |comment| %>
+        <div class="comment-card">
+          <h4 class="comment-header">Comments</h4>
+          <p class="comment-card-text"><%= comment.author.name%>: <%= comment.text %></p>
+        </div>
+      <% end %>
+    </div>
+  </div>
+  <%= button_tag(:type => 'button') do %>
+    <% link_to "See all posts", user_posts_path %>
+  <% end %>
+</div>
```

Comment:
- [ ] Kindly style your design to match the given wireframes as this is a core requirement. Styling is not important but the position of elements in the given [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAACCR4VL4LBTMV4QHNVY6KHNUZDOHN3A) has to be followed.

Given design
![image](https://github.com/LeslieAine/blog-app1/assets/81932478/2afa246f-d5f5-4107-a2e3-e03f71b9a4c3)

Your design
![image](https://github.com/LeslieAine/blog-app1/assets/81932478/95e847a1-f0f9-4043-88b5-f00f7172ec86)

#### Notice that the comments are separated by a border from the post in the given wireframe